### PR TITLE
[IDB Ignore] [MDB Ignore] Wallening Revert

### PR DESCRIPTION
**Pull request by actioninja**
# About this PR
image

I drew the short straw and got stuck making the initial move on a complicated revert.

The plan of action that has been discussed is going to roughly look like this:
- rebase every merge past wallening on to the commit before wallening, wallening related prs excluded
- pr this branch
- once this is merged, pull out everything that can live comfortably on main in to separate prs (map helper changes, offset code with it set to 0, split vis code with nothing using it, etc)
- rebase wallening back on to master with these changes

This is the "rebase to pre-wallening" PR. It's still a work in progress and likely has significant merge skew.

:cl:
del: Reverts wallening. The codebase has been rolled back to a pre-wallening version, and non-wallening related changes reapplied on top. The changelog prior to this entry may not be entirely accurate.
/:cl:

### CODE REVIEW

**Comment by atlasle** [comment]
why is this happening?

**Comment by norsvenska** [comment]
> why is this happening?

image

**Comment by CliffracerX** [comment]
Watching this PR with great interest.  This was the right call for the server, even if a painful one.

**Comment by Timonkeyn** [comment]
the un-wallening

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -243,10 +243,8 @@
 
 /datum/emote/living/jump/run_emote(mob/living/user, params, type_override, intentional)
 	. = ..()
-
-	var/original_transform = user.transform
-	animate(user, transform = user.transform.Translate(0, 4), time = 0.1 SECONDS, flags = ANIMATION_PARALLEL)
-	animate(transform = original_transform, time = 0.1 SECONDS)
+	animate(user, pixel_y = user.pixel_y + 4, time = 0.1 SECONDS)
+	animate(pixel_y = user.pixel_y - 4, time = 0.1 SECONDS)
```
Comment (Ghommie) [inline code/modules/mob/living/emote.dm:247]: I'd still like this to use pixel_z because of the SIDE_MAP map format.

**Review by Ghommie — CHANGES_REQUESTED** [review]
Not bad! I thought this would have been a much messier process but apparently it is not.

**Comment by SyncIt21** [comment]
superseded by #86161

**Pull request closed without merging**